pkg/kustomize/kustomize_cli_test.go

```diff
@@ -0,0 +1,63 @@
+package kustomize_test
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/testhelpers"
+	"github.com/jenkins-x/jx/pkg/kustomize"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+)
+
+func TestKustomizeCLI_FindKustomize_HasKustomize(t *testing.T) {
+	origJXHome, testJXHome, err := testhelpers.CreateTestJxHomeDir()
```

Comment:
I don't think that you need to create a tmp JX_HOME directory here. Something like this should do:

```
	tmpDir, err := ioutil.TempDir("", "TestKustomizeCLI_FindKustomize")
	require.NoError(t, err, "failed to create temporary directory")
	defer func() {
		_ = os.RemoveAll(tmpDir)
	}()
```

Technically _ testJXHome_ is also "just" a temporary directory, but it is also "more", so a reader of this test might get confused on why a tmp _JX_HOME_ directory should be required.